code/_expts/slotnick_etal_2001/analyze/stats/de_StaticizerCC.m

```diff
@@ -3,93 +3,76 @@
 
 for ti=1:2
     if (ti==1), ds='train', else ds='test'; end;
-    
+
     stats.perf.(ds) = cell(size(mss));
-    
+
     for mi=1:length(mss)
         ms    = mss{mi};
-        
+
         p     = [ms.p];
         o    = [p.output];
-        
+
         tmp   = de_calcPErr( vertcat(o.(ds)), mSets.data.test.T, 2);
-        
-        if (strcmp(mSets.data.taskType, 'categorical'))
-            switch(mSets.data.stimSet) % Use TLBL?
-                case 'blob-dot',
-                    trial_types = {'on', 'off'};
-                case 'plus-minus',
-                    trial_types = {'right', 'left'};
-                otherwise, error('Unimplemented stimSet for categorical. Choose from {blob-dot}');
-            end
-        else
-            switch(mSets.data.stimSet)
-                case 'blob-dot',
-                    trial_types = {'near', 'far'};
-                case 'paired-squares',
-                    trial_types = {'same', 'different'};
-                case 'plus-minus',
-                    trial_types = {'near', 'far'};
-                otherwise, error('Unimplemented stimSet for coordinate. Choose from {blob-dot, paired-squares}');
-            end
-        end
+        trial_types = {'easy', 'hard'};
```

Comment:
Every experiment had an `easy` or `hard` type trial type, and these were separated in the analysis later. So, I migrated the stats code to analyze that, as it's more meaningful in the paper.